src/funfuzz/js/compile_shell.py

```diff
@@ -715,6 +773,25 @@ def obtainShell(shell, updateToRev=None, updateLatestTxt=False):  # pylint: disa
         os.remove(shell.getS3TarballWithExtFullPath())
 
 
+def set_build_options(shell, opts, revision):
```

Comment:
I think this should remain in `__init__()` if you can make `CompiledShell.main()` either `staticmethod` or `classmethod`